scipy/stats/_continuous_distns.py

```diff
@@ -362,6 +362,17 @@ def fit(self, data, **kwds):
 
         return loc, scale
 
+    def _munp(self, n):
+        """
+        @returns Moments of standard normal distribution for integer n >= 0
+
+        See https://arxiv.org/pdf/1209.4340.pdf
+        """
+        if  n % 2 == 0:
```

Comment:
there is more than 1 space here after the `if` causing `pyflakes` to fail in the CI builds. 

ref:
https://travis-ci.org/scipy/scipy/jobs/658946985#L565

you can run a linter locally to check your changes before pushing via: 
`pycodestyle <filename>` assuming of course you have pycodestyle available in your dev environment.